simpledotorg/simple-server
# Added colors to BP Controlled, BP Uncontrolled, and Missed Visits %s

Added colors to BP Controlled, BP Uncontrolled, and Missed Visits percentages to make the tables easier to scan.

**Story card:** None

## Because

When all of the percentages are purple, it's hard to scan tables to see who is performing well and who is not.

## Test instructions

Shouldn't break anything outside of the tables it's meant to affect. Click around My Facilities pages, click around Reports tables, and formatting of % pills should be colored appropriately instead of purple.

Note, the values are relatively arbitrary based on how we think IHCI is valuing performance. It's rough.
